test/subview_test.js

```diff
@@ -66,14 +66,22 @@ function testChildMixin(ChildMixin) {
           spyOn(subview, 'remove');
         });
 
-        it('should correctly just provided subview', () => {
+        it('should correctly remove just provided subview', () => {
           view.destroySubviews(secondSubview);
 
           expect(secondSubview.remove).toHaveBeenCalled();
           expect(subview.remove).not.toHaveBeenCalled();
           expect(subviewCount(view)).toBe(1);
         });
 
+        it('should not destroy any views if `null` was provided', () => {
```

Comment:
Plus rename spec to non-view instance instead of `null`, which would be more correct